packages/teams-js/src/public/meeting.ts

```diff
@@ -170,49 +170,90 @@ export namespace meeting {
   /**
    * Property bag for the meeting reaction received event
    *
+   * @hidden
+   * Hide from docs.
+   *
+   * @internal
+   * Limited to Microsoft-internal use
+   *
    * @beta
    */
   export interface MeetingReactionReceivedEventData {
     /**
      * Indicates the type of meeting reaction received
+     *
+     * @hidden
```

Comment:
I don't think you need to have the hidden tag on each member if the containing interface is marked as hidden, feel free to remove these inner ones